src/App.tsx

```diff
@@ -1,15 +1,42 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
+interface Movie {
+  title: string;
+  description: string;
+  imgUrl: string;
+  imdbUrl: string;
+  imdbId: string;
+}
+
+const movies: Movie[] = moviesFromServer;
+
+const filteredMovies = (allMovies: Movie[], query: string): Movie[] => {
```

Comment:
A small adjustment with naming for this helper function may help here.
It's a function, so it's suppose to do something, thus, instead of naming it `filteredMovies` I'd prefer `filterMovies` naming. 

It will just make more sense inside your component: `const filteredMovies = filterMovies(...)`, don't you think so?